examples/webgl_materials_envmaps_hdr.html

```diff
@@ -252,20 +255,28 @@
 
 					standardMaterial.roughness = params.roughness;
 					standardMaterial.bumpScale = - 0.05 * params.bumpScale;
+
 					var newEnvMap = standardMaterial.envMap;
+
 					switch( params.envMap ) {
+
 						case 'None': newEnvMap = null; break;
-						case 'LDR': newEnvMap = ldrCubeRenderTarget; break;
-						case 'HDR': newEnvMap = hdrCubeRenderTarget; break;
-						case 'RGBM16': newEnvMap = rgbmCubeRenderTarget; break;
+						case 'LDR': newEnvMap = ldrCubeRenderTarget ? ldrCubeRenderTarget.texture : undefined; break;
+						case 'HDR': newEnvMap = hdrCubeRenderTarget ? hdrCubeRenderTarget.texture : undefined; break;
+						case 'RGBM16': newEnvMap = rgbmCubeRenderTarget ? rgbmCubeRenderTarget.texture : undefined; break;
```

Comment:
[I usually avoid assigning `undefined`, too](https://github.com/mrdoob/three.js/pull/8787/files#diff-82c186ef67cf396f2895cf4d8ddd7b15R233). When I was refactoring this example a couple of weeks ago, I initially did assign `null`, but then the envMap didn't get rendered. I now tried assigning `null` again and it works just fine. I think that the issue I encountered back then must have had to do with the PMREM files.

Anyway, I went a little bit further while adjusting this example and moved the logic out of the `render` loop. As you can see [here](https://github.com/vanruesc/three.js/blob/rendertarget-texture/examples/webgl_materials_envmaps_hdr.html#L254-L282), the dat.GUI parameters are evaluated in there. Would you prefer a solution that avoids that or do you like the current setup (just with `null` instead of `undefined`)?